service/src/main/kotlin/fi/espoo/evaka/s3/ContentTypeUtils.kt

```diff
@@ -9,55 +9,58 @@ import java.io.InputStream
 
 val tika: org.apache.tika.Tika = org.apache.tika.Tika()
 
-enum class ContentType(val value: String) {
-    JPEG("image/jpeg"),
-    PNG("image/png"),
-    PDF("application/pdf"),
-    MSWORD("application/msword"),
-    MSWORD_DOCX("application/vnd.openxmlformats-officedocument.wordprocessingml.document"),
-    OPEN_DOCUMENT_TEXT("application/vnd.oasis.opendocument.text"),
-    TIKA_MSOFFICE("application/x-tika-msoffice"),
-    TIKA_OOXML("application/x-tika-ooxml"),
-    VIDEO_ANY("video/*"),
-    AUDIO_ANY("audio/*"),
-}
+enum class ContentTypePattern(
+    private val type: String,
+    private val subtypePattern: String,
+    private val allowedFileExtensions: Set<String>
+) {
+    JPEG("image", "jpeg", setOf("jpg", "jpeg")),
+    PNG("image", "png", setOf("png")),
+    PDF("application", "pdf", setOf("pdf")),
+    MSWORD("application", "msword", setOf("doc")),
+    MSWORD_DOCX("application", "vnd.openxmlformats-officedocument.wordprocessingml.document", setOf("docx")),
+    OPEN_DOCUMENT_TEXT("application", "vnd.oasis.opendocument.text", setOf("odt")),
+    TIKA_MSOFFICE("application", "x-tika-msoffice", setOf("doc", "docx")),
+    TIKA_OOXML("application", "x-tika-ooxml", setOf("doc", "docx")),
+    VIDEO_ANY("video", "*", setOf("avi", "mp4", "mpeg", "ogv", "webm", "3gp")),
```

Comment:
Unrelated to the changes in this PR, but has `mov` (`video/quicktime`) been left out intentionally here? It is a common media container format for videos taken on iOS devices and also in post-production.

Reply:
Probably not, added it